admin/config.yml

```diff
@@ -0,0 +1,23 @@
+backend:
+  name: github
+  repo: jduss4/elizabethzaleski
+  branch: main
+  site_domain: https://mysite.com
+  base_url: https://mysite.com
+  auth_endpoint: /api/auth
+media_folder: assets/cms
+public_folder: assets/cms/
+collections:
+  - label: "Pages"
+    name: "pages"
+    files:
+    - name: 'info'
+      label: 'Info'
+      identifier_field: 'name'
+      file: "./_data/info.json"
+      fields:
+        - { label: 'Main Heading', name: 'title', widget: 'string' }
```

Comment:
You can have whatever fields you want here; see [Widgets](https://decapcms.org/docs/widgets/) in Decap docs